### ISSUE TITLE
Teams & browsers keyboard shortcut conflict (important message)

### ISSUE THREAD

**Issue by IncandescentChrysalis** [IsmaelMartinez/teams-for-linux#1389]
**Is your feature request related to a problem? Please describe.**
`Ctrl + Shift + I` is the shortcut for important messages (still working, despite that feature having disappeared from message formatting: go figure… M$ as usual?).

However, it conflicts with browsers' shortcut for console (queries inspection).
One of the other behaviour might be triggered in the same context.

For my own experience, that shortcut in a new message marks it as important.
However, replying in a conversation initially triggers console, and after a few tries, triggers the "important message" feature.

**Describe the solution you'd like**
I am undecided on what course of action would be best.
On one hand, following Teams shortcut might be best as part of that application, but on the other hand, since in the end we are in a modified browser, following their "standard" might be nice too.
Both features have value.

**Additional notes**
Would there be other cases of conflict?

**Comment by IsmaelMartinez** [comment]
Hi @IncandescentChrysalis , This sounds a bit complicated. We do allow for disabling shortcuts (see disableGlobalShortcuts in the [config options](https://github.com/IsmaelMartinez/teams-for-linux/tree/develop/app/config)), but disabling it depending on what is triggering I don't even know how possible would it be. I would have a think but it might not be possible, specially if it is something that is not documented (so they might be removing)

**Comment by IsmaelMartinez** [comment]
closing due inactivity. Hopefully the workaround to disable some global shoutcuts in the app does the job for you. Come back with more information if you want the issue/enhancement to be re-open. I close issues on a regular basis to keep the backlog manageable.

**Comment by IncandescentChrysalis** [comment]
I guess the real closing reason here is not inactivity (I reported everything interesting I had as input), but rather the lack of foreseable solution.
From what I understand, it comes from the difficulty to handle keyboard shortcuts betweens the ones defined by the glorified web browser Electron is, and the ones defined by the Teams application directly.

**Comment by IsmaelMartinez** [comment]
Happy to reopen this but I never got a reply from you and need to close issues that get stale, as otherwise I can't focus on much.

We use the web version shortcuts and you can disable shortcuts at the moment, but not much else.

**Comment by IsmaelMartinez** [comment]
re-reading this and I wonder if the shortcut is also still available for the browser, but with another shortcut version. The shortcuts in browsers are different than the shortcuts in the app, and this wrapper uses the browser shortcuts. I will check if I find it.

**Comment by IsmaelMartinez** [comment]
Ok, I just found where they have moved this. You can find it under the + -> 'Set delivery Option' 

<img width="316" alt="image" src="https://github.com/user-attachments/assets/f1b73dd0-d31c-4b40-b170-64edc7071aa8">

The shortcut is the one you mentioned `Ctrl + Shift + I` and I have test it on my side without having any side effects (seems to be working as expected), but I am on a Mac and the shortcut for dev tools is different. 

Did you try disabling the shortcut (see disableGlobalShortcuts in the [config options](https://github.com/IsmaelMartinez/teams-for-linux/tree/develop/app/config)) ? If you do that, what do you get?

**Comment by IncandescentChrysalis** [comment]
I just tested the shortcut again writing a message, and that triggered the "Important message" marker, without popping the developer tools this time.
If I set my context outside of a message writing box, it pops those tools.

Maybe is it fine in the end? Did I inadvertently lose focus when I wrote the messages leading to this issue? This does not sound like myself, but I cannot reproduce the problem as of now.

Disabling the general shortcuts would cut me out of those developer tools, especially the network tab, which is always nice to have available, isn't it? :smile_cat:
